src/BigBlueButton.php

```diff
@@ -90,6 +92,38 @@ public function getApiVersion()
         return new ApiVersionResponse($xml);
     }
 
+    /**
+     * Check if connection to api can be established with the baseurl and secret
+     * @return bool connection successful
+     */
+    public function checkConnection(): bool
```

Comment:
For a good UX it's important for me to tell the user why his request failed, so in my opinion it would be great if that would be possible. Maybe this can also be split. Have a static method to get the exact reason and this to report if it's working. Personally I would also prefer a name prefixed with `is`, but that's not that important.

Reply:
@sualko @Yurujai Have a look at my lastest commits? Is it more like what you had in mind?